test/conformance/chainsaw/cli/two-results-with-namespace/policy.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  annotations:
+    pod-policies.kyverno.io/autogen-controllers: none
+  name: test-jmespath
```

Comment:
This should also have a `namespace` field just with a different value other than `default` according to the test case.